RELEASENOTES.md

```diff
@@ -4,6 +4,15 @@ This file contains highlights and announcements covering all components.
 For more details see `CHANGELOG.md` files maintained in the root source
 directory of each individual package.
 
+## Unreleased
+
+* Added `net9.0` targets to all the components.
+
+* Removed `net6.0` targets from all the components.
```

Comment:
IMO worth to add more explicit note that this version is no longer supporting .net6.

Reply:
how about this one?
Removed `net6.0` targets from all components, as `.NET 6` will reach end of life on November 12, 2024.